contracts/auction-handler/FastLaneAuctionHandler.sol

```diff
@@ -519,18 +416,29 @@ contract FastLaneAuctionHandler is FastLaneAuctionHandlerEvents, Ownable, Reentr
     |             Modifiers             |
     |__________________________________*/
 
-    modifier whenNotPaused() {
-        if (paused) revert RelayPermissionPaused();
+    modifier onlyCurrentValidator() {
+        require(msg.sender != address(0), "msg.sender = 0 address");
+        require(msg.sender == block.coinbase || msg.sender == validatorsDataMap[block.coinbase].payee, "only this block's validator");
         _;
     }
 
-    modifier onlyEOA() {
-        if (msg.sender != tx.origin) revert RelayPermissionSenderNotOrigin();
+    modifier onlyActiveValidators() {
+        require(msg.sender != address(0), "msg.sender = 0 address");
```

Comment:
```suggestion
```
This should be effectively impossible (implies someone found the keys to the zero address), I don't think we need to check for this

Reply:
@Elyx0 thoughts?

Reply:
Ben comment is true, msg.sender will never be address(0)